packages/config/config/devices/0x0344/he-ht01.json

```diff
@@ -0,0 +1,795 @@
+					"value": 6
+				},
+				{
+					"label": "Time regulator and Floor sensor",
+					"value": 7
+				}
+			]
+		},
+		"12": {
+			"label": "Air Temp Min",
+			"description": "Allowable range: 1 - 36, x10",
+			"unit": "0.1 °C",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 360,
+			"defaultValue": 210
+		},
+		"13": {
+			"label": "Air Temp Max",
+			"description": "Allowable range: 2 - 37, x10",
+			"unit": "0.1 °C",
+			"valueSize": 2,
+			"minValue": 20,
+			"maxValue": 370,
+			"defaultValue": 270
+		},
+		"14": {
+			"label": "Floor Temp Min",
+			"description": "Allowable range: 1 - 36, x10",
+			"unit": "0.1 °C",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 360,
+			"defaultValue": 180
+		},
+		"15": {
+			"label": "Floor Temp Max",
+			"description": "Allowable range: 2 - 37, x10",
+			"unit": "0.1 °C",
+			"valueSize": 2,
+			"minValue": 20,
+			"maxValue": 370,
+			"defaultValue": 320
+		},
+		"16": {
+			"label": "Floor Temp Calibration",
+			"description": "Allowable range: -10 - 10, x10",
+			"unit": "0.1 °C",
+			"valueSize": 1,
+			"minValue": -100,
+			"maxValue": 100,
+			"defaultValue": 0
+		},
+		"17": {
+			"label": "Air Temp Calibration",
+			"description": "Allowable range: 10 - 10, x10",
+			"unit": "0.1 °C",
+			"valueSize": 1,
+			"minValue": -100,
+			"maxValue": 100,
+			"defaultValue": 0
+		},
+		"18": {
+			"label": "Temp Hysteresis",
+			"description": "Allowable range: 0.2 - 10, x10",
+			"unit": "0.1 °C",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 100,
+			"defaultValue": 5
+		},
+		"19": {
+			"label": "Time Correction by Controller",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"20": {
+			"label": "Time Format",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "24 hour",
+					"value": 0
+				},
+				{
+					"label": "12 hour (AM / PM)",
+					"value": 1
+				}
+			]
+		},
+		"21": {
+			"label": "Week Day Override",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 7,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Monday",
+					"value": 1
+				},
+				{
+					"label": "Tuesday",
+					"value": 2
+				},
+				{
+					"label": "Wednesday",
+					"value": 3
+				},
+				{
+					"label": "Thursday",
+					"value": 4
+				},
+				{
+					"label": "Friday",
+					"value": 5
+				},
+				{
+					"label": "Saturday",
+					"value": 6
+				},
+				{
+					"label": "Sunday",
+					"value": 7
+				}
+			]
+		},
+		"22": {
+			"label": "Time Override",
+			"unit": "HHMM",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 2359,
+			"defaultValue": 0
+		},
+		"23": {
+			"label": "Time Regulation Reg On Time",
```

Comment:
Please no abbreviations that require a manual to explain them. What's a "Reg On Time"?

Reply:
Added description and fixed label